cgi-bin/dashboard-svn.cgi

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/perl -T
+# ${license-info}
+# ${developer-info
+# ${author-info}
+# ${build-info}
+
+use CGI;
+
+my $repo = "file:///var/www/svn/scdb";
+
+my $q = new CGI;
+
+my $limit = $q->param('limit') ? $q->param('limit') : '10';
+
+$limit =~ /(\d+|all)/;
+$limit = $1;
+
+$ENV{PATH}="/bin:/usr/bin:/sbin:/usr/bin:/usr/sbin";
+
+my $output;
+
+if($limit eq 'all') {
+  $output = `/usr/bin/sudo /usr/bin/svn log --xml $repo`;
```

Comment:
no backticks. build up a command array and use `CAF::Process`
e.g.
```perl
my @command = qw(/usr/bin/sudo /usr/bin/svn log --xml);
push(@command, '-l', $limit, '-r', 'HEAD:1') if ($limit ne 'all');
push(@command, $repo);

my $p = CAF::Process(\@command);
$ouput = $p->output();
# do something with $? exitcode in case of failure
```